src/Sparrow/ByteString.cs

```diff
@@ -809,7 +809,7 @@ private ByteString AllocateExternal(byte* valuePtr, int size, ByteStringType typ
             {
                 if (_externalCurrentLeft == 0)
                 {
-                    var tmp = Math.Min(16 * Constants.Size.Megabyte, _allocationBlockSize * 2);
+                    var tmp = Math.Min(2 * Constants.Size.Megabyte, _allocationBlockSize * 2);
                     AllocateExternalSegment(tmp);
```

Comment:
No biggy here, we are reserving less (which is essentially what I am doing on the new allocators).